### PR TITLE
Add FlashTrade to Dapp list

### PR DESCRIPTION
**Pull request by Treggs6**

### CODE REVIEW

**Comment by benlvb** [comment]
@Treggs6 can you change the logo to just the icon? Can't fit the whole logo with text

File: assets/flashtrade.PNG
Comment (benlvb) [inline assets/flashtrade.PNG:1]: Could you delete this file please.

**Comment by benlvb** [comment]
Could you delete the previous logo file please.

**Comment by Treggs6** [comment]
> @Treggs6 can you change the logo to just the icon? Can't fit the whole logo with text
Changed to just logo with black bg.

**Review by benlvb — APPROVED** [review]

**Pull request merged**